test/unit/plugins/ucx/ucx_backend_test.cpp

```diff
@@ -274,24 +274,6 @@ void *releaseValidationPtr(nixl_mem_t mem_type, void *addr)
     return nullptr;
 }
 
-void
-allocateWrongGPUTest(nixlUcxEngine *ucx, int dev_id) {
-    nixlBlobDesc desc = {0};
-    nixlBackendMD* md;
-    void* buf;
-
-    allocateBuffer(VRAM_SEG, dev_id, desc.len, buf);
-
-    desc.devId = dev_id;
-    desc.addr = (uint64_t) buf;
-
-    int ret = ucx->registerMem(desc, VRAM_SEG, md);
-
-    nixl_exit_on_failure((ret == NIXL_ERR_NOT_SUPPORTED), "Failed to register memory", "test");
-
-    releaseBuffer(VRAM_SEG, dev_id, buf);
```

Comment:
@rakhmets should we reconsider this whole test with removal of CUDA dep in #946 ?